homeassistant/components/generic/config_flow.py

```diff
@@ -0,0 +1,151 @@
+"""Config flow for generic (IP Camera)."""
+
+import imghdr
+import logging
+from typing import Any, Dict
+
+import av
+import voluptuous as vol
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.camera import DEFAULT_CONTENT_TYPE
+from homeassistant.const import (
+    CONF_AUTHENTICATION,
+    CONF_NAME,
+    CONF_PASSWORD,
+    CONF_USERNAME,
+    CONF_VERIFY_SSL,
+    HTTP_BASIC_AUTHENTICATION,
+    HTTP_DIGEST_AUTHENTICATION,
+)
+
+from .camera import GenericCamera
+
+# pylint: disable=unused-import
+from .const import (
+    CONF_CONTENT_TYPE,
+    CONF_FRAMERATE,
+    CONF_LIMIT_REFETCH_TO_URL_CHANGE,
+    CONF_STILL_IMAGE_URL,
+    CONF_STREAM_SOURCE,
+    DEFAULT_NAME,
+    DOMAIN,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class GenericIPCamConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for generic IP camera."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_LOCAL_POLL
+
+    def __init__(self):
+        """Initialise the config flow."""
+        super().__init__()
+        self._errors = {}
+
+    async def _test_connection(self, info):
+        """Verify that the camera data is valid before we add it."""
+        await self.async_set_unique_id(info[CONF_STILL_IMAGE_URL])
+        try:
+            self._abort_if_unique_id_configured()
+        except data_entry_flow.AbortFlow as err:
+            if "already_configured" in str(err):
+                self._errors["base"] = "still_image_already_configured"
+                return False
```

Comment:
Check if its in `self._async_current_ids()` instead

Reply:
Fixed at c75b7ac